locale/fi/LC_MESSAGES/django.po

```diff
@@ -3574,6 +3579,74 @@ msgstr "Ei osumia"
 msgid "Search for..."
 msgstr "Etsi..."
 
+#: templates/support_channels.html
+msgid "Documentation"
+msgstr "Asiakirjat"
+
+#: templates/support_channels.html
+msgid "Aplus manual"
+msgstr "Aplus-käsikirja"
+
+#: templates/support_channels.html
+msgid "Aplus website"
+msgstr "Aplus-verkkosivusto"
+
+#: templates/support_channels.html
+msgid "Community"
+msgstr "Yhteisö"
+
+#: templates/support_channels.html
+msgid "Microsoft Teams"
+msgstr "Microsoft Teams"
+
+#: templates/support_channels.html
+msgid "Create an issue in Github"
+msgstr "Luo ongelma Githubissa"
+
+#: templates/support_channels.html
+msgid "Announcements"
+msgstr "Ilmoitukset"
+
+#: templates/support_channels.html
+msgid "Current Development"
+msgstr "Nykyinen kehitys"
+
+#: templates/support_channels.html
+msgid "Other servers that use A+"
+msgstr "Muut A + -palvelinta käyttävät palvelimet"
+
+#: templates/support_channels.html
+msgid "Exam server"
+msgstr "Tenttit palvelu"
+
+#: templates/support_channels.html
+msgid "Test server"
+msgstr "Ulkoinen palvelu"
+
+#: templates/support_channels.html
+msgid "Learning"
+msgstr "Oppiminen"
+
+#: templates/support_channels.html
+msgid "Accessible Teaching"
+msgstr "Esteetön opetus"
+
+#: templates/support_channels.html
+msgid "Exam studio with Aplus"
+msgstr "Exam studio with Aplus"
```

Comment:
Not translated at all yet. The EXAM system name is capitalized like you see here: https://e-exam.fi/in-english/

Finnish: "{brand}-tehtävät EXAM-tentissä"